tests/test_validations.py

```diff
@@ -0,0 +1,150 @@
+"""
+Test cases for the 'validations' module
+"""
+
+import json
+import os
+import pathlib
+import unittest.mock
+from copy import copy
+from typing import Dict
+from typing import Generator
+
+import pystac
+import pytest
+import requests
+
+from stac_api_validator import validations
+
+
+@pytest.fixture
+def r_session() -> Generator[requests.Session, None, None]:
+    yield requests.Session()
+
+
+@pytest.fixture
+def catalog_dict() -> Generator[Dict[str, str], None, None]:
+    current_path = pathlib.Path(os.path.dirname(os.path.abspath(__file__)))
+
+    with open(current_path / "resources" / "landing_page.json") as f:
+        # Load the contents of the file into a Python dictionary
+        data = json.load(f)
+
+    yield data
+
+
+@pytest.fixture
+def sample_item() -> Generator[pystac.Item, None, None]:
+    current_path = pathlib.Path(os.path.dirname(os.path.abspath(__file__)))
+
+    with open(current_path / "resources" / "sample-item.json") as f:
+        # Load the contents of the file into a Python dictionary
+        data = json.load(f)
+
+    yield pystac.Item.from_dict(data)
+
+
+@pytest.fixture
+def expected_headers() -> Generator[Dict[str, str], None, None]:
+    yield {
+        "User-Agent": "python-requests/2.32.3",
+        "Accept-Encoding": "gzip, deflate",
+        "Accept": "*/*",
+        "Connection": "keep-alive",
+        "Authorization": "api-key fake-api-key-value",
+    }
```

Comment:
Instead of hard-coding the `requests` version you could add this import:
```python
from importlib.metadata import version
```
Then load the version dynamically:

```suggestion
@pytest.fixture
def expected_headers() -> Generator[Dict[str, str], None, None]:
    requests_version = version("requests")
    yield {
        "User-Agent": f"python-requests/{requests_version}",
        "Accept-Encoding": "gzip, deflate",
        "Accept": "*/*",
        "Connection": "keep-alive",
        "Authorization": "api-key fake-api-key-value",
    }
```